packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPageLayout.js

```diff
@@ -0,0 +1,88 @@
+import { Box } from 'grommet'
```

Comment:
This file is modeled off the styling elements for page layout in app-content-pages.